libs/core/src/repo/mod.rs

```diff
@@ -57,6 +59,16 @@ impl CoreV3 {
 
                 tx.drop_safely()?;
             }
+
+            drop(source);
+
+            let mut path = PathBuf::from(&config.writeable_path);
+            path.push("lockbook_core__repo__schema_v2__CoreV2");
+            drop(remove_file(path));
+
+            let mut path = PathBuf::from(&config.writeable_path);
+            path.push("lockbook_core__repo__schema__CoreV1");
+            drop(remove_file(path));
```

Comment:
Reason for `drop`? Is this some trick / idiomatic practice on an `io::Result<()>` instead of just `let _ = `?
If not, I'd include a comment why.

Reply:
oh I just think it's prettier, I've seen it reasonably often

Reply:
[30b2521](https://github.com/lockbook/lockbook/pull/1589/commits/30b2521aeffaeb73355649146fc14e5905684261)